src/components/Customer/Profile/CustomerProfile.jsx

```diff
@@ -11,17 +11,20 @@ const CustomerProfile = ({ setViewValue, gId }) => {
   const [customerId, setCustomerId] = useState(1);
   const [view, setView] = useState('Home');
   const [data, setData] = useState({
-    user_name: '',
+    user_name: ''
   });
 
   useEffect(() => {
+    // console.log("customer profile", gId)
     fetch(`/db/customer/gId/${gId}`, {
```

Comment:
move this function to its own method and then call it inside of useEffect

Reply:
I'm not sure who's working on this code.

Reply:
ok, if its not your code, then don't touch it for now, but let's make a note about it and put a refactor ticket in the backlog